api/db-cdr/generate-cdr/test-data/pfhh_skip.sql

```diff
@@ -0,0 +1,94 @@
+--this query helps build the prep_pfhh_non_answer_insert table
+--it contains all skip combos for each question
+INSERT INTO `all-of-us-ehr-dev.brian_test.prep_pfhh_non_answer_insert`
```

Comment:
Is this intentional?

Reply:
this is SQL used to create the insert table. It's manual SQL that I had to run per skip answer combos. This is located in the test scripts. I just added it incase we have to revisit this again.